packages/executor/src/services/EventsService/versions/0.0.7.ts

```diff
@@ -125,7 +125,36 @@ export class EntryPointV7EventsService {
     this.LAST_BLOCK_KEY = `${this.chainId}:LAST_PARSED_BLOCK:${this.entryPoint}`;
   }
 
+  async pollEvents(publicClient: PublicClient) {
+    let blockNumber: bigint;
+    const currentBlockNumber = await this.publicClient.getBlockNumber();
```

Comment:
What happens if getBlockNumber fails? Can we add a try/catch block to handle errors?